packages/file_selector/file_selector/example/lib/open_image_page.dart

```diff
@@ -15,8 +15,11 @@ class OpenImagePage extends StatelessWidget {
       label: 'images',
       extensions: <String>['jpg', 'png'],
     );
-    final List<XFile> files =
-        await openFiles(acceptedTypeGroups: <XTypeGroup>[typeGroup]);
+    final List<XFile> files = await openFiles(
+      acceptedTypeGroups: <XTypeGroup>[typeGroup],
+      initialDirectory: '/',
+      confirmButtonText: 'select image file',
```

Comment:
I can't think of any OS where all-lowercase is the style for button labels. This is also probably not actually a better label than the default OS label, so I think it's best to just not show it in the example; it's likely to be rarely used.